src/GitHub.App/Services/ModelService.cs

```diff
@@ -125,8 +126,12 @@ IObservable<IEnumerable<AccountCacheItem>> GetUserOrganizations()
         public IObservable<IReadOnlyList<IRepositoryModel>> GetRepositories()
         {
             return GetUserRepositories(RepositoryType.Owner)
-                .Take(1)
-                .Concat(GetUserRepositories(RepositoryType.Member).Take(1))
+                .PublishLast()
```

Comment:
That's because I'm getting into the habit of returning connectable subscriptions that auto-dispose themselves when I'm returning blobs of data that one or more callers can subscribe to, so that's where my brain went to on this one. I'm replacing all of these methods with a single repository query on another branch, so I didn't put a lot of thought into it. `PublishLast` is probably overkill here. We could use `LastAsync` or we could do `TakeLast` here as well.